websites/M/Minesweeper/presence.ts

```diff
@@ -57,12 +63,16 @@ presence.on('UpdateData', async () => {
         presenceData.details = 'Viewing the best players'
         break
       }
+      case '/championship': {
+        presenceData.details = 'Viewing championship standings'
+        break
+      }
       case '/chat': {
         presenceData.details = 'Chatting'
         break
       }
       case '/equipment': {
-        presenceData.details = 'Viewing their equipment'
+        presenceData.details = 'Viewing owned equipments'
```

Comment:
The plural of 'equipment' is 'equipment'
```suggestion
        presenceData.details = 'Viewing owned equipment'
```